frontend/src/routes/report/websocketMethods.tsx

```diff
@@ -0,0 +1,28 @@
+import { reportResponse } from './types.ts';
+import { notifications } from '@mantine/notifications';
+
+/**
+ * Handles incoming WebSocket messages and displays notifications based on the message status.
+ *
+ * @param {MessageEvent} event - The WebSocket message event.
+ */
+export const receiveMessage = (event: MessageEvent) => {
```

Comment:
I think it should be named "handle message", since the receiving is done by the WebSocket class, we're just handling